help/en/releasenotes/current-draft-note.shtml

```diff
@@ -238,9 +238,13 @@
                 <li></li>
             </ul>
 
-        <h4>Lenz</h4>
+        <h4>Lissy</h4>
             <ul>
-                <li></li>
+                <li>Updated the Lissy Reporter support, including
+                improved display of the messages. Note that this
+                has changed the format of the reports, so users
+                who have written scripts to use this might need to
+                check and update them.</li>
             </ul>
 
```

Comment:
The Lizzy reporter is an Uhlenbrock product. So this needs to go under the `Uhlenbrock` section. Also, `Lenz` is a German manufacturer so I think it would be good to keep the Lenz section, unless it has been a long time since it was used.

https://www.uhlenbrock.de/de_DE/produkte/loconet/index.htm
https://www.lenz-elektronik.de/index.php